src/views/Dissolutions/ReviewConfirmDissolution.vue

```diff
@@ -349,65 +369,76 @@ export default class ReviewConfirmDissolution extends Mixins(DateMixin) {
     })
   }
 
-  private getHeaderNumber (sectionName): number {
```

Comment:
I didn't even know this existed 😁

Reply:
You must have blinked! Development on this has been fast and furious recently.

Reply:
Indeed!